<launch>

  <group ns='/robot_00/markers_0'>
    <node name='marker_controller' pkg='exploration' type='marker_controller.py' />
    <node name='marker_planner' pkg='exploration' type='marker_planner.py' />
  </group>

</launch>
